doc/src/build/package-upgrades.md

```diff
@@ -0,0 +1,283 @@
+        "digest": String("6R39f68p4tGqJWJTakKCyL8tz2w2XTvJ3Mu5nGwxadda"),
+    },
+    Object {
+        "type": String("published"),
+        "packageId": String("<ORIGINAL-PACKAGE-ID>"),
+        "version": Number(1),
+        "digest": String("FrBhLF2Rn4jP3SUsss7aXqwDDRtoKxgGbPm8eVkH7jrQ"),
+        "modules": Array [
+            String("sui_package"),
+        ],
+    },
+    Object {
+        "type": String("created"),
+        "sender": String("<PUBLISHER-ID>"),
+        "owner": Object {
+            "AddressOwner": String("<PUBLISHER-ID>"),
+        },
+        "objectType": String("0x2::package::UpgradeCap"),
+        "objectId": String("<UPGRADE-CAP-ID>"),
+        "version": Number(7),
+        "digest": String("BoGQ63r27DFZDMC8p7YwRcDpToFpbZ9rG1R4o4uXkaUw"),
+    },
+    Object {
+        "type": String("created"),
+        "sender": String("<PUBLISHER-ID>"),
+        "owner": Object {
+            "AddressOwner": String("<PUBLISHER-ID>"),
+        },
+        "objectType": String("<ORIGINAL-PACKAGE-ID>::sui_package::<MODULE-NAME>"),
+        "objectId": String("<PUBLISHER-ID>"),
+        "version": Number(7),
+        "digest": String("BC3KeuATKJozLNipbUz2GWzoDXbodXH4HLm59TxJSmVd"),
+    },
+]
+----- Balance changes ----
+Array [
+    Object {
+        "owner": Object {
+            "AddressOwner": String("<PUBLISHER-ID>"),
+        },
+        "coinType": String("0x2::sui::SUI"),
+        "amount": String("-9328480"),
+    },
+]
+```
+
+The result includes an **Object changes** section with two pieces of information you need for upgrading, an `UpgradeCap` ID and your package ID. 
+
+You can identify the different objects using the `Object.objectType` value in the response. The `UpgradeCap` entry has a value of `String("0x2::package::UpgradeCap")` and the `objectType` for the package reads `String("<PACKAGE-ID>::sui_package::<MODULE-NAME>")`
+
+To make sure your other packages can use this package as a dependency, you must update the manifest (Move.toml file) for your package to include this information. 
+
+Update the alias address and add a new `published-at` entry in the `[package]` section, both pointing to the value of the on-chain ID:
+
+```toml
+[package]
+name = "sui_package"
+version = "0.0.0"
+published-at = "<ORIGINAL-PACKAGE-ID>"
+
+[addresses]
+sui_package = "<ORIGINAL-PACKAGE-ID>"
+```
+
+The `published-at` value allows the Move compiler to verify dependencies against on-chain versions of those packages.
+
+After a while, you decide to upgrade your `sui_package` to include some requested features. Before running the `upgrade` command, you need to edit the manifest again. In the `[addresses]` section, you update the `sui_package` address value to `0x0` again so the validator issues a new address for the upgrade package. You can leave the `published-at` value the same, because it is only read by the toolchain when publishing a dependent package. The saved manifest now resembles the following:
```

Comment:
validator - Since this is a specific node type is there a different work to use here?